mu/interface.py

```diff
@@ -850,6 +850,19 @@ def enable(self, sibling):
         self.setAcceptDrops(True)
         sibling.setAcceptDrops(True)
 
+    def show_confirm_overwrite_dialog(self):
+        msg = QMessageBox()
+        msg.setIcon(QMessageBox.Information)
+        msg.setText("File already exists; overwrite it?")
+        msg.setWindowTitle("File already exists")
+        msg.setStandardButtons(QMessageBox.Ok | QMessageBox.Cancel)
```

Comment:
I can see why that would be more productive, but it's probably better to stay constant on default behaviour and have all message boxes with the same, no?